src/main/java/com/robosh/ejournal/data/dto/student/SaveStudentDto.java

```diff
@@ -0,0 +1,45 @@
+package com.robosh.ejournal.data.dto.student;
+
+import com.robosh.ejournal.data.dto.UserDto;
+import com.robosh.ejournal.data.entity.Parent;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class SaveStudentDto extends UserDto {
+
+    private Long groupId;
+
+    private Long schoolId;
+
+    private String password;
+
+    private List<Parent> parents;
+
```

Comment:
it's better to use List of parent ids

Reply:
i don`t think we need it, as parents are not inputed when you save student, i need this field only for Mapper. And if there is List<Long>, i will need to do that stupid code injection via String again for list which is always empty

Reply:
We can set student parents when we create them or later when update. To set them it's better to use only id, not all entity

Reply:
won`t we have other method when we update student ?

Reply:
We use SaveStudentDto both for updates and save.